components/style/values/generics/text.rs

```diff
@@ -128,3 +128,14 @@ impl<N, L> LineHeight<N, L> {
         LineHeight::Normal
     }
 }
+
+/// A generic value for the `text-overflow` property.
+#[derive(Clone, Debug, Eq, MallocSizeOf, PartialEq, ToCss)]
+pub enum TextOverflowSide {
```

Comment:
Hmm, though now that I realize of this, this is not a `generic` value per se (those are supposed to actually have generic arguments and such). Usually when the computed and specified values are the same, we just define it in the `specified` module and re-export it if needed from the `computed` one.

Of course this can be fixed in a followup if you want. Thanks for working on this btw!

Reply:
No problem! I moved `TextOverflowSide` to the `specified` module, is that all that is needed?